pkg/localstorev2/storer.go

```diff
@@ -202,6 +202,13 @@ func initDiskRepository(basePath string, opts *Options) (storage.Repository, io.
 		}
 	}
 
+	txStore := leveldbstore.NewTxStore(store)
```

Comment:
We dont need this as this is read only operation. We can use the store directly here.